scripts/xe-enable-ipv6

```diff
@@ -30,8 +30,19 @@ if [ $1 = "enable" ]; then
 	echo "IPV6_AUTOCONF=NO" >> /etc/sysconfig/network
 	chkconfig ip6tables on
 
-	echo "net.ipv6.conf.all.disable_ipv6=0" > /etc/sysctl.d/91-net-ipv6.conf
-	echo "net.ipv6.conf.default.disable_ipv6=0" >> /etc/sysctl.d/91-net-ipv6.conf
+	{
```

Comment:
I think a better way to create a file with specific content would be to use a heredoc:

```
cat > /etc/sysctl.d/91-net-ipv6.conf <<EOF
net.ipv6.conf.all.disable_ipv6=0
net.ipv6.conf.default.disable_ipv6=0
net.ipv6.conf.all.addr_gen_mode=1
net.ipv6.conf.default.addr_gen_mode=1
EOF

```

Likewise below.

Reply:
I can do it. But I need to remove indentation right? So it will look like:
```
        ....
        echo "IPV6_AUTOCONF=NO" >> /etc/sysconfig/network

        cat > /etc/sysctl.d/91-net-ipv6.conf <<EOF
echo "net.ipv6.conf.all.disable_ipv6=1"
echo "net.ipv6.conf.default.disable_ipv6=1"
EOF

        echo "IPv6 disabled.  You may now need to reboot the host"
        ...
```
If it is ok for you I will do that no worries.

Reply:
You can use `cat > /etc/sysctl.d/91-net-ipv6.conf <<-EOF` (notice the dash) and use tabs as indentation - which then will be disregarded and the output would be flush left.